crates/frontend/src/components/monaco_editor.rs

```diff
@@ -37,22 +41,35 @@ pub fn monaco_editor(
             minimap_settings.set_enabled(Some(false));
             editor_settings.set_language(Some(language.to_string().as_str()));
             editor_settings.set_automatic_layout(Some(true));
-            editor_settings.set_value(Some(data_rs.get().as_str()));
+            editor_settings.set_font_family(Some("ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, \"Liberation Mono\", \"Courier New\", monospace"));
+            logging::log!("Monaco Init value {}", data);
+            editor_settings.set_value(Some(data.as_str()));
+            // editor_settings.set_value(Some(data_rs.get().as_str()));
```

Comment:
This might break functions UI

Reply:
Oh, i will make changes wherever this component is being used to utilize this implementation